### ISSUE TITLE
Legion Prof: Missing Measurements

### ISSUE THREAD

**Issue by samuelkgutierrez** [StanfordLegion/legion#58]
@streichler I'm seeing the following failed assertion get triggered due to missing measurements.

OS: OS X
Where: Master branch

```
./cgsolver -hl:prof 1 -cat legion_prof -level 2
# some output...
Ax = A * x is done.
r = b - Ax is done.
Assertion failed: (response.has_measurement< Realm::ProfilingMeasurements::OperationMemoryUsage>()), function process_results, file /Users/samuel/devel/legion/runtime/legion_profiling.cc, line 552.
Abort trap: 6
```

**Comment by streichler** [comment]
Should be fixed with e80b39d.

**Comment by samuelkgutierrez** [comment]
Looks like that did it. Thanks, @streichler
